release-notes/5.0/preview/api-diff/preview2/.Net/5.0-preview2_System.Text.Json.md

```diff
@@ -0,0 +1,237 @@
++        public JsonArray();
++        public JsonArray(IEnumerable<bool> values);
++        public JsonArray(IEnumerable<byte> values);
++        public JsonArray(IEnumerable<decimal> values);
++        public JsonArray(IEnumerable<double> values);
++        public JsonArray(IEnumerable<short> values);
++        public JsonArray(IEnumerable<int> values);
++        public JsonArray(IEnumerable<long> values);
++        public JsonArray(IEnumerable<sbyte> values);
++        public JsonArray(IEnumerable<float> values);
++        public JsonArray(IEnumerable<string> values);
++        public JsonArray(IEnumerable<JsonNode> values);
++        public JsonArray(IEnumerable<ushort> values);
++        public JsonArray(IEnumerable<uint> values);
++        public JsonArray(IEnumerable<ulong> values);
++        public int Count { get; }
++        public bool IsReadOnly { get; }
++        public JsonNode this[int idx] { get; set; }
++        public override JsonValueKind ValueKind { get; }
++        public void Add(JsonNode value);
++        public void Clear();
++        public override JsonNode Clone();
++        public bool Contains(JsonNode value);
++        public JsonArrayEnumerator GetEnumerator();
++        public int IndexOf(JsonNode item);
++        public void Insert(int index, JsonNode item);
++        public int LastIndexOf(JsonNode item);
++        public bool Remove(JsonNode item);
++        public int RemoveAll(Predicate<JsonNode> match);
++        public void RemoveAt(int index);
++        void System.Collections.Generic.ICollection<System.Text.Json.JsonNode>.CopyTo(JsonNode[] array, int arrayIndex);
++        IEnumerator<JsonNode> System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode>.GetEnumerator();
++        IEnumerator System.Collections.IEnumerable.GetEnumerator();
++    }
++    public struct JsonArrayEnumerator : IDisposable, IEnumerator, IEnumerator<JsonNode> {
++        public JsonArrayEnumerator(JsonArray jsonArray);
++        public JsonNode Current { get; }
++        object System.Collections.IEnumerator.Current { get; }
++        public void Dispose();
++        public bool MoveNext();
++        void System.Collections.IEnumerator.Reset();
++    }
++    public sealed class JsonBoolean : JsonNode, IEquatable<JsonBoolean> {
++        public JsonBoolean();
++        public JsonBoolean(bool value);
++        public bool Value { get; set; }
++        public override JsonValueKind ValueKind { get; }
++        public override JsonNode Clone();
++        public override bool Equals(object obj);
++        public bool Equals(JsonBoolean other);
++        public override int GetHashCode();
++        public static bool operator ==(JsonBoolean left, JsonBoolean right);
++        public static bool operator !=(JsonBoolean left, JsonBoolean right);
++        public override string ToString();
++    }
+     public readonly struct JsonElement {
++        public bool IsImmutable { get; }
```

Comment:
Uh, what is this? it is not part of `System.Text.Json` ref.
The doc page for such property is empty https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.isimmutable.

@layomia @steveharter

Reply:
This was part of the Writable DOM feature we just removed.  See https://github.com/dotnet/core/pull/4506#discussion_r402696393

Reply:
Ah I see, I though it was a leftover from a separate feature.